rfc8446.xml

```diff
@@ -1227,8 +1227,8 @@ legacy_compression_method fields have the same meaning.
 However, for convenience we discuss "HelloRetryRequest" throughout this
 document as if it were a distinct message.</t>
 
-<t>The server's extensions MUST contain "supported_versions",
-otherwise, the server SHOULD send only the extensions necessary for the
+<t>The server's extensions MUST contain "supported_versions".
+The server SHOULD send only the extensions necessary for the
```

Comment:
I see what you're doing here. I would suggest perhaps

```
"supported_versions". Additionally, it SHOULD contain the
minimum set of extensions necessary for the client to generate..."
```

Reply:
SGTM.  (You own the pull request, so push away...)